packages/origin-backend/src/pods/mail/mail.service.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { MailerService, ISendMailOptions } from '@nestjs-modules/mailer';
+
+interface IIndividualMandrillMailSendStatus {
+    email: string;
+    status: 'sent';
+    _id: string;
+    reject_reason: string;
+}
+
+interface IMandrillMailSendStatus {
+    messageId: string;
+    accepted: IIndividualMandrillMailSendStatus[];
+    rejected: IIndividualMandrillMailSendStatus[];
+}
+
+@Injectable()
+export class MailService {
+    private readonly logger = new Logger(MailService.name);
+
+    constructor(private readonly mailerService: MailerService) {}
+
+    async send(sendMailOptions: ISendMailOptions) {
+        try {
+            const result: IMandrillMailSendStatus = await this.mailerService.sendMail(
+                sendMailOptions
+            );
+
+            if (result.messageId) {
```

Comment:
What does this case means? We silently skipping it. Maybe should be logged?

Reply:
Also, possibly better as 

```
if (!result.messageId) {
 return true
}
```

Reply:
This case means if `result.messageId` then we're probably using Mandrill transport. Email transporters in nodemailer seem to have different responses unfortunately, and this needs to be abstracted away,